blocks/ads-block/features/ads/ad-mapping.js

```diff
@@ -30,6 +30,13 @@ const adMapping = {
     dimensionsArray: [sz300x250, sz300x250, sz300x250],
     ampDimensionsArray: sz300x250,
   },
+  '300x250_gallery': {
```

Comment:
looks good, vaguely remember a proptypes warning. not sure if that's part of ac https://github.com/WPMedia/fusion-news-theme-blocks/issues/723

Reply:
You will see that warning if you've not set the prop correctly. We should look at that in other PR